api/jpo-conflictvisualizer-api/src/main/java/us/dot/its/jpo/ode/api/services/ReportService.java

```diff
@@ -97,66 +115,274 @@ public class ReportService {
     @Autowired
     ReportRepository reportRepo;
     
+public class ConnectionOfTravelData {

```

Comment:
Class Declaration for ConnectionOfTravelData should be done in its own file named ConnectionOfTravelData.java